lib/bolt_server/transport_app.rb

```diff
@@ -53,9 +55,21 @@ def initialize(config)
       # This is needed until the PAL is threadsafe.
       @pal_mutex = Mutex.new
 
+      @apply_task = construct_apply_task
       super(nil)
     end
 
+    def construct_apply_task
```

Comment:
Should this go in `libexec`?

Given the apply_helpers task are basically all the same code (initializing a isolated puppet env and unpacking plugin tarball) I think for bolt-server I would like to just have a single task and add logic to modify behavior.